docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,33 +263,37 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
 . Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
-
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Select **ES|QL**. 
+. Write an aggregating or non-aggregating type query:
 +
-[esql]
------
+TIP: Click the **ES|QL reference** button (image:images/esql-ref-button.png[Click the ES|QL reference button,20,20]) to the right of the query bar to open an ES|QL syntax reference guide.
++
+** **Aggregating query**: Uses {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate alert data. An alert is generated for each segment of the query, but is not associated with a specific source event since aggregating queries process data from multiple alert documents. As a result, alerts from aggregating queries do not contain source event data.
++
+Example aggregating query: 
++
+[source,esql]
+----
 FROM logs*
 | STATS count = COUNT(host.name) BY host.name
 | SORT host.name
```

Comment:
@vitaliidm I was thinking we could frame these examples in a more descriptive way. Maybe something like:

For example, if you wanted to create an alert for < describe scenario >, you can write a query like this:

Reply:
We can describe this query as: 

> Rule with the following query generates alert for each host.name that encountered more than 20 times during rule interval.

For more production oriented rules, I think we would need to contact TRADE team. But, it can be very complex, bussiness specific rule. I a m not sure we want to add it in our docs, that should have simple enough examples to give basic understanding how rules work

Reply:
Thanks for the description! That was the main item I needed for this part :)